WindSpigot-API/src/main/java/org/bukkit/ChatColor.java

```diff
@@ -340,7 +340,7 @@ public static String translateAlternateColorCodes(char altColorChar, String text
 	 * @return Any remaining ChatColors to pass onto the next line.
 	 */
 	public static String getLastColors(String input) {
-		String result = "";
+		StringBuilder result = new StringBuilder();
```

Comment:
For cases likes these, in which it's only appended once, StringBuilder is slower

Reply:
ah my bad missed that one